doc/whats_new/v0.23.rst

```diff
@@ -62,6 +62,11 @@ Changelog
   :pr:`17985` by :user:`Alan Butler <aldee153>` and
   :user:`Amanda Dsouza <amy12xx>`.
 
+- |Fix| Fixed a bug in
+  :func:`decomposition.PCA.fit_transform` which has problem that
+  PCA returns highly inaccurate results when number of features is large.
+  :pr:`21109` by `Bin Chen, Wei Guo, Lin Zeng`.
```

Comment:
can you format properly the name of the user as in the other entries?
This entry needs to be placed in a `v1.0.rst` in for the future bug fix release 1.0.1

Reply:
> can you format properly the name of the user as in the other entries?
> This entry needs to be placed in a `v1.0.rst` in for the future bug fix release 1.0.1

I will modify it further based on your comments. Thanks for your suggestion.